datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -23,6 +23,11 @@
 log = logging.getLogger('test_downloader')
 IntegrationMetadata = namedtuple("IntegrationMetadata", ["version", "root_layout_type"])
 
+# Integrations released for the last time by a revoked developper but not shipped anymore.
+EXCLUDED_INTEGRATIONS = [
+    "datadog-docker-daemon",
+]
```

Comment:
This integration was last released by a revoked key a few months ago. This integration was removed from `integrations-core`, so it shouldn't be installable on new agents with `3.core.root.layout`

cc @trishankatdatadog

Reply:
Hmm, it's a bit brittle, isn't it, because how do you test new versions of this integration going forward? I recommend you get an approved key for `3.core.root.layout` to release a new version instead.

Reply:
This integration was removed from master (https://github.com/DataDog/integrations-core/pull/5988), it is not released anymore. It's only shipped in agent 5 so it shouldn't be installed on agent 6 or 7